aexpect/remote.py

```diff
@@ -418,6 +418,7 @@ def remote_login(client, host, port, username, password, prompt, linesep="\n",
                             status_test_command=status_test_command,
                             client=client, host=host, port=port,
                             username=username, password=password)
+    session.output_prefix = host
```

Comment:
I think this was left unchecked after some initial experimentation where I wanted to check if this could work at all. Using it in the init arguments makes sense of course. Thanks and I will improve my self-review effort in the future since it seems I am letting too many silly diffs like this one slip through.